source/app/utils/document.js

```diff
@@ -565,3 +561,5 @@ export function getPageWordsBySearch(document, pageNumber, searchQuery) {
   }, [])
   return matchingWords
 }
+
+export const getAreRedactionsOnDocument = document => !document.redactions || Object.keys(document.redactions).length === 0
```

Comment:
- can we sinmplify the naming of this function.  get and are fall under  "vebs". Could we go for a name like isDocumentRedactionPresent?
- Would it make sense to simplify the expression and just check 
```Object.keys(document.redactions).length```

Reply:
- `isDocumentRedactionPresent` would be the name of the boolean the function returns, not the function itself
- That would also work for a truthiness check but is not a boolean. I wouldn't worry about it if it was an inline function but as a function, it should return a boolean

Reply:
Agreed with you reasoning with the the second comment and partially with your 1st comment. Having two verbs in the function naming convention still seems avoidable.

Reply:
```
const areRedactionsOnDocument = getAreRedactionsOnDocument(redactions);
```
is the same as 
```
const uncheckedTodos = getUncheckedTodos(todos);
```

Rather than looking it as nouns and verbs, we should be looking at it as value descriptors and actions. `areRedactionsOnDocument` describes the value, not the action to get the value.

Reply:
I still am not on the same page but given that this is around naming conventions, I can disagree and commit :p 

Another followup on the 2nd question from my original comment. What happens when the document has no redactions or has an empty array of redactions? Wouldn't this return true (when the expectation would be to return false?)

Reply:
> Another followup on the 2nd question from my original comment. What happens when the document has no redactions or has an empty array of redactions? Wouldn't this return true (when the expectation would be to return false?)

That's a good point. I just made existing logic reusable here but we should really be checking the lengths of the arrays at each page.